src/App/Program.fs

```diff
@@ -145,6 +146,11 @@ type MissionOptions
     [<Option('o', "old-image", HelpText = "Stellar-core image to use as old-image", Required = false)>]
     member self.OldImage = oldImage
 
+    [<Option("old-image-node-count",
+             HelpText = "Number of old image nodes for mixed image load mission",
```

Comment:
oh, is it to test different combinations of old vs new images? I would argue that it's better to just hardcode the desired parameters in the mission so we don't rely on the user configuring the mission correctly (as with this parameter changing, we end up testing very different things).

Reply:
I think two missions would be better tbh, so we can hardcode the exact scenarios we want to test. And I think we can refactor some common functionality to avoid duplication.

Reply:
Sounds good, it's actually more compact if we keep both setups in the same file.